cg/meta/upload/scout/raredisease_config_builder.py

```diff
@@ -61,28 +61,33 @@ def build_load_config(self) -> RarediseaseLoadConfig:
         self.include_case_files(load_config)
         self.get_sample_information(load_config)
         self.include_pedigree_picture(load_config)
-        load_config.custom_images = self.load_custom_image_sample()
+        self.load_custom_image_sample(load_config)
         load_config.human_genome_build = GenomeBuild.hg19
         return load_config
 
-    def load_custom_image_sample(self) -> CustomImages:
+    def load_custom_image_sample(self, load_config: RarediseaseLoadConfig) -> CustomImages | None:
```

Comment:
```suggestion
    def load_custom_image_sample(self, load_config: RarediseaseLoadConfig) -> None:
```
This never returns anything anymore right?